drafts/2020-04-28-this-week-in-rust.md

```diff
@@ -19,7 +19,16 @@ If you find any errors in this week's issue, [please submit a PR](https://github
 * [A Taste of WebGPU in Firefox](https://hacks.mozilla.org/2020/04/experimental-webgpu-in-firefox/)
 * [Data Ingestion with Rust and AWS Lambda](http://jamesmcm.github.io/blog/2020/04/19/data-engineering-with-rust-and-aws-lambda/#en)
 * 🎈🎉 [Announcing Rust 1.43.0](https://blog.rust-lang.org/2020/04/23/Rust-1.43.0.html). 🎉🎈
-
+* [Embedded Rust pattern - Zero Sized References](https://ferrous-systems.com/blog/zero-sized-references/)
+* [Implementing a linear Quadtree in Rust](https://snorrwe.onrender.com/posts/morton-table/)
+* [Data Ingestion with Rust and AWS Lambda](https://jamesmcm.github.io/blog/2020/04/19/data-engineering-with-rust-and-aws-lambda/#en)
+* [Memory efficient serialization of tagged union](https://robinmoussu.gitlab.io/blog/post/binary_serialisation_of_enum/)
+* [Unpacking Serde](https://www.reddit.com/r/rust/comments/g6ubuv/unpacking_serde_a_series_of_presentations_i_made/)
+* [Rust Stream: Ownership, Closures, and Threads - Oh My!](https://www.youtube.com/watch?v=2mwwYbBRJSo)
+* [Crust of Rust: Lifetime Annotations](https://docs.google.com/spreadsheets/d/15pqsOlwc2eBXNRV0GJP7Taa3NnFi5kMFpmyVerONsf8/edit#gid=853276561)
+* [Arch Linux announces independent verification of binary packages with rebuilderd (a rust application!)](https://lists.reproducible-builds.org/pipermail/rb-general/2020-April/001905.html)
```

Comment:
While cool (and this is definitely news), this link is an announcement, rather than an exploratory blog post. I generally lean away from including crate/application releases because there are so many of them. To cover this content, I'd rather see an exploration of how rebuilderd is implemented in Rust, rather than only an announcement of its existence

Reply:
Yes I agree. I will also use: https://github.com/emberian/this-week-in-rust/pull/1210#issuecomment-620202041 for further posts regarding a crate/application release.